src/App.js

```diff
@@ -3,14 +3,40 @@ import React, { Component } from "react";
 import Bookings from "./Bookings";
 import Restaurant from "./Restaurant";
 import "./App.css";
+import Heading from "./Heading";
+import TouristInfoCards from "./TouristInfoCards";
 
+import Footer from "./Footer";
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">CYF Hotel</header>
+        <Heading />
+
+        <TouristInfoCards
```

Comment:
This component name implies that it shows multiple cards but it only shows one. I would probably rename your existing component to `TouristInfoCard` (no "s" at the end) and then create another component named `TouristInfoCards` which renders all 3 of the cards.